sql/analyzer/analyzer.go

```diff
@@ -136,7 +144,11 @@ func (ab *Builder) Build() *Analyzer {
 // Analyzer analyzes nodes of the execution plan and applies rules and validations
 // to them.
 type Analyzer struct {
+	// Whether to log various debugging messages
 	Debug       bool
+	// Whether to output the query plan at each step of the analyzer
+	Verbose     bool
+	debugCtx    []string
```

Comment:
Reading the code below I had to scroll back up to see what debugCtx was.  I read Ctx and think context.Context.  Feel like another name would clear up some confusion.

Reply:
Done